src/Entity/Item.php

```diff
@@ -21,7 +23,7 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-class Item extends Entity implements StatementListHolder {
+class Item implements Comparable, EntityDocument, FingerprintHolder, StatementListHolder {
```

Comment:
Does not matter a lot yes. However putting the least important one before the by far most important one is kinda odd.